tools/build.md

```diff
@@ -29,25 +29,39 @@ within the manylinux2014 container are encouraged.
 
 We use the TFX manylinux docker that includes a working Bazel installation. Run
 the following command to start the docker (might require superuser permissions):
-```
+
+```sh
 ./tools/start_compile_docker.sh
 ```
+
 Within the docker, run the following command
+
+```sh
+PYTHON_VERSION=<version> ./tools/build_manylinux_all.sh
 ```
-./tools/build_manylinux.sh
+
+where `<version>` is one of `38`, `39`, `310`, `311`.
+
+Or, in case you want to build for all supported Python versions
```

Comment:
I've had weird caching issues with build_manylinux_all.sh end of last week, where bazel wound up building an incompatible package. I don't think it's worth diving deeper into it and instead just remove the build_manylinux_all.sh script since it does not provide that much extra value. WDYT?

Reply:
agreed! will do